src/corner_kick/src/modules/Visualizer/Canvas.tsx

```diff
@@ -0,0 +1,117 @@
+import * as PIXI from 'pixi.js';
+
+import * as storage from 'electron-json-storage';
+import * as _ from 'lodash';
+import * as Viewport from 'pixi-viewport';
+import * as React from 'react';
+
+import { IResizeEntry, ResizeSensor } from '@blueprintjs/core';
+import { Field } from '~/modules/Visualizer/components/Field';
+
+/**
+ * The visualizer object. Consists of a WebGL enabled canvas, a 2D camera, and
+ * the various objects to visualize (aka. field, ball, robots, etc.)
+ */
+export class Canvas extends React.Component {
+  public state = {
+    width: 0,
+    height: 0,
+  };
+
+  private canvasRef: React.RefObject<HTMLCanvasElement>;
+  private app: PIXI.Application;
+  private viewport: Viewport;
+
+  constructor(props: {}) {
+    super(props);
+
+    // A React ref allows us to access the HTML node once it is created.
+    this.canvasRef = React.createRef();
+  }
+
+  public componentDidMount() {
+    const canvas = this.canvasRef.current!;
+
+    // Create a PIXI.js app. PIXI.js acts as an abstraction layer above the WebGL context.
+    // TODO make background color parameterizable.
+    this.app = new PIXI.Application({
+      autoStart: true,
+      antialias: true,
+      backgroundColor: 0x417f11,
+      height: 0,
+      width: 0,
+      view: canvas,
+    });
+
+    // Create the 2D camera
+    // TODO make world size parameterizable
```

Comment:
`TODO`'s should be tied to github issues

Reply:
> `TODO`'s should be tied to github issues

The TODO was completed by moving the constant to the constant file